conf/tools/install.sh

```
@@ -25,6 +25,123 @@ install_pip() {
+}
+
+python_cflags_need_redhat_rpm_config() {
+    $1 - <<'PY' 2>/dev/null | grep -qE 'redhat|^-specs=| -specs='
+import sysconfig
+print(sysconfig.get_config_var("CFLAGS") or "")
+PY
+}
+
+verify_python_c_extension_build() {
+    local python_bin=$1
+    local tmp_c=`mktemp /tmp/zstackctl-build-check.XXXXXX.c`
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)